backend/apps/accounts/views.py

```
@@ -861,6 +861,70 @@ def get(self, request):
+
+    def post(self, request):
+        serializer = AvatarUploadSerializer(data=request.data)
+        
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        
+        avatar = serializer.validated_data['avatar']
+        
+        # Optimize image
+        try:
+            img = Image.open(avatar)
```

Ruff (0.15.20)

[error] Undefined name `Image`

(F821)